scripts/Source/sslConfigMenu.psc

```diff
@@ -715,7 +748,12 @@ event OnSelectST()
 			ActorLib.MakeNoStrip(ItemRef)
 		endIf
 		SetTextOptionValueST(GetStripState(ItemRef))
-		ForcePageReset()
+	
+		string StripEditorPossibilityST = "StripEditorPossibility_"+Options[1]+"_"+Options[2]
+		SetSliderOptionValueST(StorageUtil.GetIntValue(ItemRef, "SometimesStrip", 100), "{0}%", false, StripEditorPossibilityST)
+		SetOptionFlagsST(SexLabUtil.IntIfElse(ActorLib.IsAlwaysStrip(ItemRef), OPTION_FLAG_NONE, OPTION_FLAG_DISABLED), false, StripEditorPossibilityST)
```

Comment:
refresh the value and flag just for the linked option avoiding the use of the ForcePageReset() and the amount of time wasted on the full inventory.